fusedwind/turbine/test/data/DTU10MW.dp3d

```diff
@@ -1,4 +1,5 @@
 # web00  web01  web02  web03
+# top  mid  mid  mid
```

Comment:
I think adding a header is a good solution, i.e. the future fusedwind version:
`# v0.3.0`?
 Then we can still leave the web naming user defined since we are parsing the first line for the above pattern.
Should the header be added to all input files?